roles/common/apache2/apache2_install/templates/apache2.conf.j2

```diff
@@ -0,0 +1,45 @@
+# Include module configuration:
```

Comment:
Are these first 10 lines necessary? They don't appear to be in the original apache2.conf files.